tools/bump-version.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
```

Comment:
We should clarify the goal of this script.What version is it bumping: the module's of the provider's withi the module examples?